src/NRediSearch/Suggestion.cs

```diff
@@ -0,0 +1,108 @@
+﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
+using System;
+using StackExchange.Redis;
+
+namespace NRediSearch
+{
+    public class Suggestion
```

Comment:
suggest `sealed` if not known to need inheritance

Reply:
I actually wonder whether this should be `readonly struct`, but... meh, that might limit us if `JRediSearch` gets exotic in the future